sim/rogue/rogue.go

```diff
@@ -9,8 +9,9 @@ import (
 )
 
 const (
-	SpellFlagBuilder     = core.SpellFlagAgentReserved2
-	SpellFlagFinisher    = core.SpellFlagAgentReserved3
+	SpellFlagBuilder     = core.SpellFlagAgentReserved1
+	SpellFlagFinisher    = core.SpellFlagAgentReserved2
+	SpellFlagPoison      = core.SpellFlagAgentReserved3
```

Comment:
We'd like to make a core poison spell flag so that we can apply the 20% poison DR correctly. Is that something that would work in place of this rogue-specific flag?

Reply:
It would be checked here https://github.com/wowsims/sod/blob/80f0d0f0db6d721b9baf25a21d2995c2cc6b2912/sim/core/spell_result.go#L518-L520 and afaik would apply to hunter's Serpent Sting and Viper Sting

Reply:
Semantically, it's more of a DeadlyBrew flag. If the `PoisonDamageTakenMultiplier` becomes a thing in SOD, there's neither much clash nor much synergy between this and a global flag.